fuels-core/Cargo.toml

```diff
@@ -9,10 +9,16 @@ repository = "https://github.com/FuelLabs/fuels-rs"
 description = "Fuel Rust SDK core."
 
 [dependencies]
+fuel-tx = "0.1"
 fuel-types = { version = "0.1", default-features = false }
+fuel-vm = "0.1"
```

Comment:
Noting that the verification of signatures doesn't really depend on `fuel-vm`, and should be abstracted out into a lighter crate. Can you file and issue for this in `fuel-vm`?

Reply:
Yeah, I noticed that too. It *shouldn't* depend on the VM, but it does as of now because the signing/verification functions live there and reimplementing it here isn't a good idea. 

In the future, yes, this should 100% be extracted out, I'll file an issue as soon as I can!